lisa/operating_system.py

```diff
@@ -233,6 +233,18 @@ def _install_packages(self, packages: Union[List[str]]) -> None:
         )
 
 
+class Fedora(Linux):
```

Comment:
Can you update RedHat to inherit from Fedora? If dnf is compatible from RedHat 6.x, the `_install_packages` of RedHat can be removed.

Reply:
It should be conditional for RedHat: yum for < 8, dnf otherwise. Do you guys have that conditional check infra available already?

Reply:
Version check should be added in os classes when it's necessary, but we can keep `_install_packages` in this case.